kops/kops.go

```diff
@@ -248,18 +210,18 @@ func (k *KopsCmd) GetKubeConfig(cluster clusteroperatorv1alpha1.KopsConfig) (clu
 
 	config := clusteroperatorv1alpha1.KubeConfig{}
 
-	kopsCmd := k.path +
+	kopsCmdStr := k.path +
 		" export kubecfg" +
 		" --name=" + cluster.Name +
 		" --state=" + cluster.StateStore +
 		" --kubeconfig=tmp/config-" + cluster.Name
 
-	err := utils.RunStreamingCmd(kopsCmd)
+	err := utils.RunStreamingCmd(kopsCmdStr)
 	if err != nil {
 		return clusteroperatorv1alpha1.KubeConfig{}, err
 	}
 
-	file, err := ioutil.ReadFile("tmp/config-" + cluster.Name)
+	file, err := ioutil.ReadFile(viper.GetString("tmp.dir") + "config-" + cluster.Name)
```

Comment:
change to `file, err := ioutil.ReadFile(viper.GetString("tmp.dir") + "/config-" + cluster.Name)` which adds the extra slash in front of 'config' to make the file path correct